backend/clickhouse/migrations/000098_recreate_sessions_joined_vw.down.sql

```diff
@@ -0,0 +1,13 @@
+DROP VIEW IF EXISTS sessions_joined_vw;
+CREATE VIEW IF NOT EXISTS sessions_joined_vw AS
+select ProjectID as ProjectId,
+    CreatedAt as Timestamp,
+    mapFromArrays(
+        arrayMap(x->splitByChar('_', x, 2) [2], FieldKeys),
+        arrayMap(
+            (k, kv)->substring(kv, length(k) + 2),
+            arrayZip(FieldKeys, FieldKeyValues)
+        )
+    ) as SessionAttributes,
+    *
```

Comment:
isn't this the same as the schema in the `up` migration?

Reply:
It is! Which is the same as `backend/clickhouse/migrations/000066_add_sessions_joined_vw.up.sql`. The issue is we added fields to sessions after we created the mv, so that mv doesn't contain those fields, but the logic is still sound. We did something similar to `errors_joined_vw`.

Reply:
oh makes sense, thanks!